fs/fs.go

```diff
@@ -427,7 +427,11 @@ func (fs *filesystem) Mount(ctx context.Context, mountpoint string, labels map[s
 		for _, s := range src {
 			sociDesc, ok := c.imageLayerToSociDesc[s.Target.Digest.String()]
 			if !ok {
-				rErr = fmt.Errorf("unable to resolve layer %q from %q, mounting entire layer using overlayfs: %w", s.Target.Digest, s.Name, snapshot.ErrNoZtoc)
+				log.G(ctx).WithFields(logrus.Fields{
+					"layerDigest": s.Target.Digest.String(),
+					"image":       s.Name.String(),
+				}).Infof("skipping mounting layer as FUSE mount: %v", snapshot.ErrNoZtoc)
+				rErr = fmt.Errorf("skipping mounting layer %s as FUSE mount: %w", s.Target.Digest.String(), snapshot.ErrNoZtoc)
 				continue
```

Comment:
nit: if there is more than 1 source, this message will be logged multiple times, right? Should we just break here?